[FE-1329] Launch Hibana

shrirupa opened the pull request:
FE-1329 - Launch Hibana

### What Changed

- Set the isHibanaEnabled value in HibanaContext to true
- Removed Theme Toggle from Profile page
- Removed cypress tests that passed for OG only
- Removed IS_HIBANA_ENABLED check
- Had to upgrade enzyme-adapter-react-16" to  "^1.15 since there were some randomly failing unit test
     - As a result had to upgrade snapshots



### How To Test
- its easier to review this PR with Hide white spaces turned on - https://github.com/SparkPost/2web2ui/pull/2015/files?diff=split&w=1
- Check if Hibana is the default and only theme of the app
- User should not be able to toggle back and forth between the OG and Hibana theme 

### To Do

- [ ] Address Feedback

nicklemmon commented on src/pages/blocklist/components/tests/IncidentsCollection.test.js:
```diff
@@ -36,7 +36,7 @@ describe('Blocklist Component: IncidentsCollection', () => {
     expect(queryByText('spammy mcspamface')).toBeInTheDocument();
     expect(queryByText('Dec 3 2019 at 10:00am')).toBeInTheDocument();
-    expect(queryByText('Last 7 Days')).toBeInTheDocument();
+    // expect(queryByText('Last 7 Days')).toBeInTheDocument(); TODO: figure out why this failing
```
Oddly enough I'm not seeing where this would have been rendering to begin with.

```js
const columns = [
  { label: 'Resource', sortKey: 'resource' },
  { label: 'Blocklist', sortKey: 'blocklist_name' },
  { label: 'Date Listed', sortKey: 'occurred_at' },
  { label: 'Date Resolved', sortKey: 'resolved_at' },
  { label: '' },
];
```

I don't see a `dateRange` column anywhere.

nicklemmon commented on src/pages/blocklist/tests/WatchlistAddPage.test.js:
```diff
@@ -42,16 +42,16 @@ describe('WatchlistAddPage', () => {
+  it.skip('submits the resource and redirects on save', () => {
     const resource = 'test';
     const promise = Promise.resolve({ resource });
     const watchlistAdd = jest.fn(() => promise);
```
Changing line 61 to the following appears to fix this one:
```js
const save = getByRole('button', { name: saveText });
```

shrirupa replied:
😄  Thanks for the fix!

nicklemmon commented on src/pages/templates/components/tests/SavedIndicator.test.js:
```diff
@@ -3,7 +3,7 @@ import { mount } from 'enzyme';
 import TestApp from 'src/__testHelpers__/TestApp';
```
Honestly I think it's fine to just delete this test - I don't think it's terribly valuable.

nicklemmon reviewed (changes requested):
@shrirupa Very glad to see this green! I left a few comments here and there.

Additionally, I would go ahead and remove the OG-specific steps from our CircleCI config. Oddly enough the "OG" integration tests are just the Hibana tests at this point so no need to keep them both running. If you would like help doing that let me know.

shrirupa commented on cypress/tests/integration/dashboardV2/dashboardPageV2.spec.js:
```diff
@@ -1,4 +1,4 @@
-import { IS_HIBANA_ENABLED, USERNAME } from 'cypress/constants';
```
good point!

shrirupa commented:
@nicklemmon  Thanks for the feedback and fixes! I have addressed all the comments (need to rebase, this PR will have rebase required in spec file till the day we deploy 😅 ). I will need help removing turning off OG tests in circle ci, and is that something we do after we deploy ?

nicklemmon commented on cypress/tests/integration/navigation/navigation.spec.js:
```diff
@@ -0,0 +1,750 @@
+import { USERNAME } from 'cypress/constants';
+
+describe('the Hibana navigation', () => {
```
This can just be 'the navigation' - no need to call it 'the Hibana navigation'

nicklemmon commented on cypress/tests/integration/dashboard/dashboardPage.spec.js:
```diff
@@ -0,0 +1,718 @@
+
+const PAGE_URL = '/dashboard';
+
+describe('Version 2 of the dashboard page', () => {
```
Same as the navigation spec - this can just be 'The dashboard page' instead of referencing version 2.

nicklemmon reviewed (changes requested):
@shrirupa Some very nit-picky points of feedback at this point. No need to address now - it can wait until a future rebase prior to the release.

nicklemmon reviewed (approved):
@shrirupa This looks good. We'll need to rebase and re-review before launch.
